addons/hr_maintenance/views/maintenance_views.xml

```diff
@@ -26,7 +26,13 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='owner_user_id']" position="replace">
                 <field name="employee_id" string="Created By"
-                       options="{'no_create_edit': True, 'no_open': True}"/>
+                       widget="many2one_employee"
```

Comment:
actually instead we can put `many2one_avatar_employee` instead

Reply:
Done